aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,94 @@ uid: mvc/controllers/actions
 
 Controllers, actions, and action results are a fundamental part of how developers build apps using ASP.NET Core MVC.
 
-## What is a Controller
+## What is a Controller?
 
-In ASP.NET MVC, a *Controller* is used to define and group a set of actions. An *action* (or *action method*) is a method on a controller that handles incoming requests. Controllers provide a logical means of grouping similar actions together, allowing common sets of rules (e.g. routing, caching, authorization) to be applied collectively. Incoming requests are mapped to actions through [routing](routing.md).
+In ASP.NET MVC, a controller is used to define and group a set of actions. An action (or action method) is a method on a controller which handles incoming requests. Controllers provide a logical means of grouping similar actions together, allowing common sets of rules, such as routing, caching, and authorization, to be applied collectively. Incoming requests are mapped to actions through [routing](xref:mvc/controllers/routing).
 
-In ASP.NET Core MVC, a controller can be any instantiable class that ends in "Controller" or inherits from a class that ends with "Controller". Controllers should follow the [Explicit Dependencies Principle](http://deviq.com/explicit-dependencies-principle) and request any dependencies their actions require through their constructor using [dependency injection](dependency-injection.md).
+In ASP.NET Core MVC, a controller is an instantiable class in which at least one of the following conditions is true:
+* The class name is suffixed with "Controller"
+* The class inherits from a class whose name is suffixed with "Controller"
 
 By convention, controller classes:
+* Reside in the project's root-level *Controllers* folder
+* Inherit from `Microsoft.AspNetCore.Mvc.Controller`
 
-* Are located in the root-level "Controllers" folder
+Additional configuration is required if these two conventions are disregarded.
 
-* Inherit from Microsoft.AspNetCore.Mvc.Controller
+Controllers should follow the [Explicit Dependencies Principle](http://deviq.com/explicit-dependencies-principle) and request any dependencies their actions require through their constructor using [constructor injection](xref:mvc/controllers/dependency-injection#constructor-injection).
 
-These two conventions are not required.
-
-Within the Model-View-Controller pattern, a Controller is responsible for the initial processing of the request and instantiation of the Model. Generally, business decisions should  be performed within the Model.
+Within the **M**odel-**V**iew-**C**ontroller pattern, a controller is responsible for the initial processing of the request and instantiation of the model. Generally, business decisions should be performed within the model.
 
 > [!NOTE]
-> The Model should be a *Plain Old CLR Object (POCO)*, not a `DbContext` or database-related type.
+> The model should be a *Plain Old CLR Object (POCO)*, not a `DbContext` object or a database-related type.
```

Comment:
Is this note helpful?  It hardly seems likely that someone would use a DbContext as a model.